src/common/modal/modal.module.scss

```diff
@@ -24,4 +24,8 @@
   &:focus {
     outline: none;
   }
+  width: 100%;
+  max-width: units(32);
```

Comment:
I think this too restrictive as "global style", for example in ticket https://helsinkisolutionoffice.atlassian.net/browse/VEN-625 I need a wider modal than 32 units.

Reply:
That's a good point, maybe the modal should have a size prop for this?

Reply:
Could this be like before (the modal adapts to it's content size). What do you think?

Reply:
Maybe that's the best approach after all, I'll do this in a bit